vbh/build-harness-extensions/README.md

```diff
@@ -11,11 +11,11 @@ In order to use the build harness and extensions, make yourself a [token](https:
 GITHUB_USER := $(shell echo $(GITHUB_USER) | sed 's/@/%40/g')
 GITHUB_TOKEN ?=
 
--include $(shell [ -f ".build-harness-bootstrap" ] || curl -sSL -o .build-harness-bootstrap -H "Authorization: token $(GITHUB_TOKEN)" -H "Accept: application/vnd.github.v3.raw" "https://raw.github.com/open-cluster-management/build-harness-extensions/master/templates/Makefile.build-harness-bootstrap"; echo .build-harness-bootstrap)
+-include $(shell [ -f ".build-harness-bootstrap" ] || curl -sSL -o .build-harness-bootstrap -H "Authorization: token $(GITHUB_TOKEN)" -H "Accept: application/vnd.github.v3.raw" "https://raw.github.com/stolostron/build-harness-extensions/master/templates/Makefile.build-harness-bootstrap"; echo .build-harness-bootstrap)
```

Comment:
I believe `master` was renamed to `main`.

Reply:
Updated. Thanks.